app/models/comment.rb

```diff
@@ -29,7 +29,8 @@ class Comment < ActiveRecord::Base
   # I *do not* have any idea why Time.now wasn't working (I assume it was a time-zone thing), but this works:
   scope :visible, -> { where('visible_at <= ?', 0.seconds.from_now) }
   scope :undeleted, -> { where(deleted: false) }
-
+  scope :comments_of_today, ->(current_user) { where(user_id: current_user.id,
+                                       created_at: DateTime.now.in_time_zone.to_date.beginning_of_day..DateTime.now.in_time_zone.to_date.end_of_day)}
```

Comment:
Line is too long. [149/80]<br>Align the elements of a hash literal if they span more than one line.<br>Expression at 33, 149 should be on its own line.